src/test/java/uk/gov/pay/api/service/PaymentSearchServiceTest.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.pay.api.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.pay.api.app.config.PublicApiConfig;
+import uk.gov.pay.api.auth.Account;
+import uk.gov.pay.api.model.TokenPaymentType;
+
+import javax.ws.rs.client.Client;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PaymentSearchServiceTest {
```

Comment:
No happy path test?

Reply:
All happy path tests are covered in `PaymentResourceSearchITest`